client/service-worker.js

```diff
@@ -0,0 +1,41 @@
+// The Lounge - https://github.com/thelounge/lounge
+/* global clients */
+"use strict";
+
+self.addEventListener("push", function(event) {
```

Comment:
I take it this file gets distributed as is? Meaning, we don't webpack it, correct?

I noticed the syntax is a bit less ES6-looking (`var` instead of `const`, no arrow functions, etc.), and I'm assuming that's because of the above. If a client supports service workers, isn't it fair to assume it will support `const` and `=>`?

Also, is ESLint going through this file, or ignoring it?